src/Test.UnitTests.Sarif.Converters/TestData/FortifyFprConverter/ExpectedOutputs/OneResultBasic.sarif

```diff
@@ -6,6 +6,7 @@
       "results": [
         {
           "ruleIndex": 0,
+          "level": "none",
```

Comment:
>"level": "none", [](start = 10, length = 16)

i know you hate it (possibly!) see my other comment in the cs file

Reply:
You're right, I do :-). I'm going to look very carefully at this!

---
In reply to: [292263379](https://github.com/microsoft/sarif-sdk/pull/1528#discussion_r292263379) [](ancestors = 292263379)

Reply:
Ok, I looked, and I don't agree. See my comments in the converter file and the converter tests file.

---
In reply to: [292523811](https://github.com/microsoft/sarif-sdk/pull/1528#discussion_r292523811) [](ancestors = 292523811,292263379)

Reply:
hah, thanks. i will make the change! 

---
In reply to: [292553684](https://github.com/microsoft/sarif-sdk/pull/1528#discussion_r292553684) [](ancestors = 292553684,292523811,292263379)